tests/Application/app/config/config.yml

```diff
@@ -61,3 +61,6 @@ ongr_elasticsearch:
                 index_name: sylius
             mappings:
                 SyliusElasticSearchPlugin: {}
+
+sylius_elastic_search:
+    attribute_whitelist: ['MUG_COLLECTION_CODE', 'MUG_MATERIAL_CODE']
```

Comment:
Can you remove `MUG_COLLECTION_CODE` from the list, so we will check if it is really removed from API response?